command.go

```diff
@@ -248,6 +249,20 @@ func ExecuteCopy(c parser.Command, _ *VHS) {
 	_ = clipboard.WriteAll(c.Args)
 }
 
+// ExecuteEnv sets env with given key-value pair.
+func ExecuteEnv(c parser.Command, v *VHS) {
+	_ = os.Setenv(c.Options, c.Args)
+
+	if c.Options == "PROMPT" {
+		if v.Options.Shell.Env != nil {
+			v.Options.Shell.Env[0] = strings.ReplaceAll(v.Options.Shell.Env[0], ">", c.Args)
+		} else if v.Options.Shell.Command != nil {
+			last := len(v.Options.Shell.Command) - 1
+			v.Options.Shell.Command[last] = strings.ReplaceAll(v.Options.Shell.Command[last], ">", c.Args)
+		}
+	}
```

Comment:
I couldn't find a better way to set up during the evaluation time. This also feels very hacky. I would love to know a better way to do this.

Reply:
```go
package main

import (
	"fmt"
	"os"
)

// Supported shells of VHS
const (
	bash       = "bash"
	cmdexe     = "cmd"
	fish       = "fish"
	nushell    = "nu"
	powershell = "powershell"
	pwsh       = "pwsh"
	zsh        = "zsh"
)

// Shell is a type that contains a prompt and the command to set up the shell.
type Shell struct {
	Command []string
	Env     []string
}

var prompt = func() string {
	prompt := os.Getenv("PROMPT")
	if prompt == "" {
		return "> "
	}
	return prompt
}()

// Shells contains a mapping from shell names to their Shell struct.
var Shells = map[string]Shell{
	bash: {
		Env:     []string{fmt.Sprintf("PS1=\\[\\e[38;2;90;86;224m\\]%s \\[\\e[0m\\]", prompt), "BASH_SILENCE_DEPRECATION_WARNING=1"},
		Command: []string{"bash", "--noprofile", "--norc", "--login", "+o", "history"},
	},
	zsh: {
		Env:     []string{fmt.Sprintf(`PROMPT=%%F{#5B56E0}%s %%F{reset_color}`, prompt)},
		Command: []string{"zsh", "--histnostore", "--no-rcs"},
	},
	fish: {
		Command: []string{
			"fish",
			"--login",
			"--no-config",
			"--private",
			"-C", "function fish_greeting; end",
			"-C", fmt.Sprintf(`function fish_prompt; set_color 5B56E0; echo -n "%s "; set_color normal; end`, prompt),
		},
	},
	powershell: {
		Command: []string{
			"powershell",
			"-NoLogo",
			"-NoExit",
			"-NoProfile",
			"-Command",
			fmt.Sprintf(`Set-PSReadLineOption -HistorySaveStyle SaveNothing; function prompt { Write-Host '%s' -NoNewLine -ForegroundColor Blue; return ' ' }`, prompt),
		},
	},
	pwsh: {
		Command: []string{
			"pwsh",
			"-Login",
			"-NoLogo",
			"-NoExit",
			"-NoProfile",
			"-Command",
			fmt.Sprintf(`Set-PSReadLineOption -HistorySaveStyle SaveNothing; Function prompt { Write-Host -ForegroundColor Blue -NoNewLine '%s'; return ' ' }`, prompt),
		},
	},
	cmdexe: {
		Command: []string{"cmd.exe", "/k", fmt.Sprintf("prompt=^%s ", prompt)},
	},
	nushell: {
		Command: []string{"nu", "--execute", "$env.PROMPT_COMMAND = {''}"},
	},
}
```

Was trying something like this but it didn't work.